### ISSUE TITLE
I do object detection, and I should choose which model is more suitable

### ISSUE THREAD

**Issue by meijunhui1998** [OpenGVLab/InternVL#69]

**Comment by czczup** [comment]
Sorry for the late reply. 

Because during the SFT stage we only utilized the RefCOCO data (detecting a box based on a single sentence), and did not use COCO data (detecting all objects of a class), it might not perform well on object detection tasks. 

To improve object detection performance, it would be necessary to fine-tune the pre-trained model, such as InternVL-Chat-V1.2-Plus, on COCO detection data.

**Comment by czczup** [comment]
This is an example for visual grounding.

<img width="916" alt="image" src="https://github.com/OpenGVLab/InternVL/assets/23737120/f37bc593-62b9-4faf-afdb-64ed4e5afb53">

**Comment by 77h2l** [comment]
@czczup Thanks for your work.
I wonder what's the advantage of the internVL model comparing to the SAM-based visual perception model，is there any comparision? Thx
